docs/en/connector-v2/source/FakeSource.md

```diff
@@ -16,54 +16,53 @@ just for some test cases such as type conversion or connector new feature testin
-| tinyint.min         | tinyint  | no       | 0                       |
-| tinyint.max         | tinyint  | no       | 127                     |
-| tinyint.template    | list     | no       | -                       |
-| smallint.fake.mode  | string   | no       | range                   |
-| smallint.min        | smallint | no       | 0                       |
-| smallint.max        | smallint | no       | 32767                   |
-| smallint.template   | list     | no       | -                       |
-| int.fake.template   | string   | no       | range                   |
-| int.min             | int      | no       | 0                       |
-| int.max             | int      | no       | 0x7fffffff              |
-| int.template        | list     | no       | -                       |
-| bigint.fake.mode    | string   | no       | range                   |
-| bigint.min          | bigint   | no       | 0                       |
-| bigint.max          | bigint   | no       | 0x7fffffffffffffff      |
-| bigint.template     | list     | no       | -                       |
-| float.fake.mode     | string   | no       | range                   |
-| float.min           | float    | no       | 0                       |
-| float.max           | float    | no       | 0x1.fffffeP+127         |
-| float.template      | list     | no       | -                       |
-| double.fake.mode    | string   | no       | range                   |
-| double.min          | double   | no       | 0                       |
-| double.max          | double   | no       | 0x1.fffffffffffffP+1023 |
-| double.template     | list     | no       | -                       |
-| table-names         | list     | no       | -                       |
-| common-options      |          | no       | -                       |
-
-### schema [config]
-
-#### fields [Config]
-
-The schema of fake data that you want to generate
-
-#### Examples
+## Source Options
+
+|        Name         |   Type   | Required |         Default         |                                                                                      Description                                                                                      |
+|---------------------|----------|----------|-------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| schema              | config   | yes      | -                       | Define Schema information                                                                                                                                                             |
+| fields              | config   | yes      | -                       | Define Field information                                                                                                                                                              |
```

Comment:
Where is `fields`?

Reply:
![Uploading cd0ba9dc-0ce2-4c58-86ca-e2785bd38458.jpeg…]()

Reply:
```
### schema [config]

#### fields [Config]

The schema of fake data that you want to generate

#### Examples

```
here

Reply:
This is a sub config of `schema`. Not for `FakeSource` root config. Please refer https://github.com/apache/seatunnel/blob/dev/docs/en/concept/schema-feature.md

Reply:
done